src/Game/Game.elm

```diff
@@ -115,7 +115,10 @@ update msg model =
             ( { model | currentScreen = InventoryScreen }, Cmd.none )
 
         InvMsg msg ->
-            ( Inventory.update msg model, Cmd.none )
+            if msg == ExitMsg then
+                ( { model | currentScreen = MapScreen }, Cmd.none )
```

Comment:
I took your advice and I'm trying to implement as a new message from Game. But I'm having problems dealing with the type signature of the function `Inventory.view`. 

It is declared as `view : Model -> Html (InventoryMsg Drag Drop)` but if I add a `onClick` with a new exit message (type `Game.Msg`, which I moved to `Game.Data.Msg` to avoid cyclic dependencies), Elm will say that there is a type mismatch between `Model -> Html (InventoryMsg Drag Drop)` and `Model -> Html Game.Data.Msg`. I'm stuck :(